endpoints.json

```diff
@@ -64,6 +64,18 @@
           "created_at": 1604113380000
         }
       }
+    },
+    "POST /api/articles/:article_id/comments": {
```

Comment:
For this endpoint a user would want to know what they need to send to the endpoint to make a successful request. Include something like a `exampleRequest` property that shows that that looks like